v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/transforms/KafkaTransform.java

```diff
@@ -124,7 +128,7 @@ public static KafkaIO.Read<byte[], byte[]> readBytesFromKafka(
             .withConsumerConfigUpdates(config)
             .withConsumerFactoryFn(new FileAwareConsumerFactoryFn());
 
-    if (enableCommitOffsets) {
+    if (enableCommitOffsets != null && enableCommitOffsets) {
```

Comment:
As mentioned in an earlier comment, it's ok to change the parameter type from `Boolean` to `boolean` since `null` has no differentiated meaning compared to false for these methods. The caller should ensure that the provided argument is non-null. That's the case for all callers in this repository, since `KafkaReadOptions` defines a default value for the pipeline option `enableCommitOffsets` and the caller in the legacy template currently sets this to `false` unconditionally.

Reply:
Ah, I see #1901 addresses this.